config/environments/production.rb

```diff
@@ -22,15 +24,15 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
+  # Compress CSS using a preprocessor.
   config.assets.js_compressor = :uglifier
   # config.assets.css_compressor = :sass
```

Comment:
Since we're also compressing JS, what do you say we either keep the original comment here, or split the lines out like so?

```suggestion  
  # Compress JavaScript using a preprocessor.
  config.assets.js_compressor = :uglifier
  
  # Compress CSS using a preprocessor.
  # config.assets.css_compressor = :sass
```